src/pages/ProfilesExplorerView/components/SceneProfilesExplorer/ui/ExplorationTypeSelector.tsx

```diff
@@ -1,52 +1,71 @@
-import { css } from '@emotion/css';
-import { SelectableValue } from '@grafana/data';
-import { InlineLabel, RadioButtonGroup, Select, useStyles2 } from '@grafana/ui';
+import { css, cx } from '@emotion/css';
+import { GrafanaTheme2, SelectableValue } from '@grafana/data';
+import { Button, Icon, InlineLabel, useStyles2 } from '@grafana/ui';
+import { noOp } from '@shared/domain/noOp';
 import React from 'react';
 
 export type ExplorationTypeSelectorProps = {
-  layout: 'radio' | 'select';
   options: Array<SelectableValue<string>>;
   value: string;
   onChange: (newValue: string) => void;
 };
 
-export function ExplorationTypeSelector({ layout, options, value, onChange }: ExplorationTypeSelectorProps) {
+export function ExplorationTypeSelector({ options, value, onChange }: ExplorationTypeSelectorProps) {
   const styles = useStyles2(getStyles);
 
   return (
     <div className={styles.explorationTypeContainer} data-testid="exploration-types">
-      <InlineLabel width="auto">Exploration type</InlineLabel>
```

Comment:
Nit: Not really happy about this world. "Exploration" is not telling me anything. Maybe "Steps" is better?

Please do not block on this, I think we can always change that word